vision/unit_tests/test_module.py

```diff
@@ -363,10 +363,96 @@ def test_return(self):
         #   and arbitrary center value
         depth_image = np.zeros(IMAGE_SIZE, dtype=int)
         arbitrary_value = 300
-        roi = region_of_interest(depth_image, arbitrary_value, (arbitrary_value, arbitrary_value))
+        roi = region_of_interest(
+            depth_image, arbitrary_value, (arbitrary_value, arbitrary_value)
+        )
 
         self.assertIs(type(roi), np.ndarray)
 
 
+class TestModuleRoll(unittest.TestCase):
+    """
+    Testing module.get_module_roll for validity.
+    """
+
+    def test_params(self):
+        """
+        Verify can handle input types
+
+        Parameters
+        ----------
+        enclosing_region: ndarray
+            The depth image.
+        module_roll: np.float64
+            Calculated float angle.
+        """
+
+        IMAGE_SIZE = [1080, 1920, 3]
+
+        enclosing_region = np.zeros(IMAGE_SIZE, dtype=np.uint8)
+        module_roll = get_module_roll(enclosing_region)
+
+        self.assertIs(type(enclosing_region), np.ndarray)
+        self.assertIs(type(module_roll), np.float64)
+
+    def test_module_roll(self):
+        img_dir = os.path.join(gparent_dir, "vision_images/module/Feb29")
+        files = os.listdir(img_dir)
+        loc = ModuleLocation()
+
+        estimates = {
+            # FORMAT: "name_of_file" : estimated float // COULD BE WILDLY OFF, just guesses as of right now
+            "2020-02-29_15.36.15.167565": 13.5,
+            "2020-02-29_15.40.46.652448": 14,
+            "2020-02-29_15.40.48.862847": 13,
+            "2020-02-29_15.40.40.444538": 9,
+            "2020-02-29_15.40.54.564617": 10,
+        }
+
+        for current_file in estimates.keys():
+            IMAGE_SIZE = [1920, 1080]
+            # grabs and loads image file (assumes all files are in order of color1, depth1, color2, depth2, etc.)
```

Comment:
> assumes all files are in order of color1, depth1...

I'm not sure what this is meant to imply. The files don't need to be in order and since they are handpicked images they will have the format you specified.